src/main/java/com/rocketden/main/exception/UserError.java

```diff
@@ -9,7 +9,8 @@
 public enum UserError implements ApiError {
 
     INVALID_USER(HttpStatus.BAD_REQUEST, "Please provide a user with a valid nickname (1-16 characters without spaces)."),
-    NOT_FOUND(HttpStatus.NOT_FOUND, "A user with given nickname could not be found in database.");
+    NOT_FOUND(HttpStatus.NOT_FOUND, "A user with given nickname could not be found in database."),
+    ACCESS_DENIED(HttpStatus.FORBIDDEN, "Access Denied.");
```

Comment:
This is just a nit but I'd suggest reusing `RoomError.INVALID_PERMISSIONS` instead